src/services/useGames.ts

```diff
@@ -10,6 +10,26 @@ export const useGameRequests = () => {
   const { axiosLoggedRequest } = React.useContext(UserContext);
   const { village } = React.useContext(VillageContext);
 
+  const getIsGameCreateBeforePlaying = React.useCallback(
```

Comment:
Same as above, I would name this function: `getUserCreatedGamesCount `.

The count is used to determine whether or not the user created games before playing, but this is not what the function is returning. It's returning the count of games the user created.